Templates/SendGrid-Timer-CSharp/run.csx

```diff
@@ -0,0 +1,36 @@
+#r "SendGrid"
+
+using System;
+using SendGrid.Helpers.Mail;
+using Microsoft.Azure.WebJobs.Host;
+
+// The 'From' and 'To' fields are automatically populated with the values specified by the binding settings.
+//
+// You can also optionally configure the default From/To addresses globally via host.config, e.g.:
+//
+// {
+//   "sendGrid": {
+//      "to": "user@host.com",
+//      "from": "Azure Functions <samples@functions.com>"
+//   }
+// }
+public static Mail Run(TimerInfo myTimer, TraceWriter log)
+{
+    log.Info($"C# Timer trigger function executed at: {DateTime.Now}");
```

Comment:
If your scenario is a Daily Report, I'd change this log message to something relevant to the scenario, e.g. "Generating daily report for ...".